modules/appeals_api/spec/services/appeals_api/pdf_construction/generator_spec.rb

```diff
@@ -145,6 +145,28 @@
             File.delete(generated_pdf) if File.exist?(generated_pdf)
           end
         end
+
+        context 'special character verification' do
+          it 'allows certain typography characters into Windows-1252' do
+            hlr = build(:minimal_higher_level_review)
+            hlr.form_data['included'][0]['attributes']['issue'] = 'Smartquotes: “”‘’'
+            hlr.save!
+            path = described_class.new(hlr, version: 'V2').generate
+            generated_pdf = PDF::Reader.new(path)
```

Comment:
I wonder if naming this something different would help differentiate between the generated_pdf that we compare with expected_pdf. It seems like this plays a different role.